Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM node:18-alpine AS base
```

Comment:
this will be gibberish to you - plz ignore. basically, we're making our container by running different commands.